PyTorch/SpeechSynthesis/FastPitch/fastpitch/model.py

```diff
@@ -319,7 +334,7 @@ def forward(self, inputs, use_gt_pitch=True, pace=1.0, max_duration=75):
 
     def infer(self, inputs, pace=1.0, dur_tgt=None, pitch_tgt=None,
               energy_tgt=None, pitch_transform=None, max_duration=75,
-              speaker=0):
+              speaker=0, condition=0):
```

Comment:
so because this is the condition index, the default is 0 (despite no condition being n_conditions = 1 and so far there only being able to be 1 condition?)
Just making sure I understand, once again my brain is melting